src/js/utils/background.js

```diff
@@ -159,8 +165,16 @@ export const backgroundAndTextColors = (backgroundArg, textArg, theme) => {
   } else {
     backgroundColor = normalizeBackgroundColor(background, theme);
     const shade = darkContext(backgroundColor, theme);
+    let transparent;
+
+    if (backgroundColor && canExtractRGBArray(backgroundColor)) {
+      const colorArray = getRGBArray(backgroundColor);
+      if (colorArray[3] < 0.5) transparent = true;
+    }
     if (shade) {
```

Comment:
Can it ever be a shade and transparent? if so will the current logic catch that?

Reply:
if the alpha value is < 0.5 shade will be undefined so shade and transparent will never both be defined.